lib/CrEOF/Geo/String/Parser.php

```diff
@@ -57,29 +52,48 @@ class Parser
     private $nextSymbol;
 
     /**
+     * @var Lexer
+     */
+    private static $lexer;
```

Comment:
```Parser::parse()``` resets the 2 internal vars used during parsing, and ```AbstractLexer::setInput()``` resets the lexer internals. Am I missing something?